### ISSUE TITLE
Add to Google Fonts

### ISSUE THREAD

**Issue by JasonSFuller** [githubnext/monaspace#120]
Please consider submitting these fonts for inclusion in the [Google Fonts](https://github.com/google/fonts) project.

**Comment by VKrishna04** [comment]
yes so we can simply add them in one line than including a whole bunch of files

**Comment by toddjonker** [comment]
It would be nice if the closing of this ticket included a rationale for the rejection of a very reasonable request.

Is Monaspace available on any similar webfont servers?

**Comment by KTibow** [comment]
Relevant: https://github.com/google/fonts/issues/7113

**Comment by jothefiredragon** [comment]
@toddjonker  The following message is based on this answer : https://github.com/githubnext/monaspace/issues/172#issuecomment-1841668842

[JSDelivr](https://www.jsdelivr.com/) might fit your needs.
```
https://cdn.jsdelivr.net/gh/githubnext/monaspace@v1.400/fonts/Web%20Fonts/Variable%20Web%20Fonts/Monaspace%20Neon/Monaspace%20Neon%20Var.woff
```

Hope this helps!

**Comment by idan** [comment]
Hi all!

Google fonts is very particular about how they want fonts to work. Their quality standards are very prescriptive. We looked at those requirements at the time and decided it wasn't a good tradeoff of effort for value — we'd need to spend a ton of energy just to rework the fonts so that Google would be willing to add them, and we preferred to spend the budget on improving the fonts (adding language support, etc etc).

Google has a right to decide the rules for their playground, but it was too many rules for us. I don't believe we're going to revisit this as our contract with the type foundry has ended and there's no additional budget for major surgeries.

Self-hosting webfonts in 2026 is not that complicated.

**Comment by jothefiredragon** [comment]
> I don't believe we're going to revisit this as our contract with the type foundry has ended and there's no additional budget for major surgeries.

With all due respect, would this means that there'd be no more updates to any Monaspace typefaces and that all issue that is still open would remain as such?

**Comment by idan** [comment]
"Dammit Jim, I'm a software engineer not a type designer!"

I very much hope not, but the reality is that we don't have budget at the moment for another round of work from the type foundry. But never say never! It wasn't easy to persuade the holders of the purse strings to fund this in the first place, and we succeeded there. Also budgets are never forever — I've learned that patience is necessary. Case in point, the original budget for Monaspace took me something like 9 months to secure.

Things like "fix the install instructions" are within our reach until we get more funding here. "Add support for new languages" is not.

**Comment by hybridherbst** [comment]
Just out of curiosity, are [those](https://github.com/google/fonts/issues/7113#issuecomment-2835807289) the issues / quality standards you mean, or is it the RFN document?

**Comment by idan** [comment]
@hybridherbst it's been a hot second but yeah those look about right. There's a loooong document somewhere with all the specs that Google Fonts requires for inclusion but I don't remember where it is.

Part of the issue is that Monaspace is uniquely _massive_ among typefaces. 750,000 glyphs, gigabytes of font files, frozen + nerdfont variants. And the build process is incredibly complex because of all the weird shit involved in texture healing.

Not saying this to complain, but to set the context for why we ultimately decided not to pursue the process. What would be a modest/sensible ask of most fonts might involve weeks of effort for something like Monaspace. We're pushing Glyphs way outside of its happy place already. 

@rileycran @heathercran might have more subject-matter expertise to share here. But if I recall correctly, that was the high-level rationale.
